python_modules/dagster/dagster/core/execution/plan/utils.py

```diff
@@ -73,5 +76,18 @@ def solid_execution_error_boundary(error_cls, msg_fn, step_context, **kwargs):
                 original_exc_info=sys.exc_info(),
                 **kwargs,
             ) from e
+
+        except DagsterExecutionInterruptedError as ie:
```

Comment:
you could include KeyboardInterrupt in the list here, in case the callsite didn't set up the signal handler